services/ticker/internal/scraper/asset_scraper.go

```diff
@@ -75,8 +75,12 @@ func fetchTOMLData(asset hProtocol.AssetStat) (data string, err error) {
 	}
 
 	req, err := http.NewRequest("GET", tomlURL, nil)
-	req.Header.Set("User-Agent", "Stellar Ticker v1.0")
+	if err != nil {
+		err = errors.New("Invalid URL or request")
```

Comment:
Use a lower case first character because nested errors get concatenated, the Go linter will pickup on this but we don't seem to run it on CI. Also wrap the error so we can see exactly what's wrong with the URL in the caller or wherever this might be outputted if it is today or ever gets outputted in the future. You'll need to import `github.com/stellar/go/support/errors` to get the Wrap call.
```suggestion
		err = errors.Wrap(err, "invalid URL or request")
```

Reply:
Done, thanks for the suggestions! :)